membership-attribute-service/app/controllers/ExistingPaymentOptionsController.scala

```diff
@@ -43,7 +44,7 @@ class ExistingPaymentOptionsController(
       maybeUserId: Option[String],
       contactRepository: ContactRepository,
       subscriptionService: SubscriptionService,
-  ): SimpleEitherT[Map[AccountId, List[Subscription[SubscriptionPlan.AnyPlan]]]] =
+  )(implicit logPrefix: LogPrefix): SimpleEitherT[Map[AccountId, List[Subscription[SubscriptionPlan.AnyPlan]]]] =
```

Comment:
now we have to make it so everything that does logging based off a request takes an identity id as an implicit parameter, which is a bit invasive but hopefully worth it?